### PR TITLE
rpcdaemon: change constants representation

### PR DESCRIPTION
**Pull request by Sixtysixter**
Modified representation for DEFAULT_FILTER_STORAGE_SIZE and DEFAULT_MAX_FILTER_AGE constants.

### CODE REVIEW

File: silkworm/silkrpc/concurrency/context_pool.hpp [hunk 1]
```diff
@@ -130,7 +130,7 @@ class ContextPool {
     boost::asio::io_context& next_io_context();
 
   private:
-    static const std::size_t DEFAULT_POOL_STORAGE_SIZE = 0x400;
+    static const std::size_t DEFAULT_FILTER_STORAGE_SIZE = 1024;  // default filter storage size, ie max num for filters in storage
```
Comment (canepat) [inline silkworm/silkrpc/concurrency/context_pool.hpp:133]: Rename this constant accordingly to [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html)

Moreover, please move this constant in `filter_storage.hpp`

File: silkworm/silkrpc/core/filter_storage.hpp [hunk 2]
```diff
@@ -70,7 +70,7 @@ class FilterStorage {
     }
 
   private:
-    static const std::size_t DEFAULT_MAX_FILTER_AGE = 0x800;
+    static const std::size_t DEFAULT_MAX_FILTER_AGE = 600;  // lasting time for unused filters in seconds
```
Comment (canepat) [inline silkworm/silkrpc/core/filter_storage.hpp:73]: Rename this constant accordingly to [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html)

**Review by canepat — CHANGES_REQUESTED** [review]

File: silkworm/silkrpc/core/filter_storage.hpp (hunk 2 above)
Comment (Sixtysixter) [reply silkworm/silkrpc/core/filter_storage.hpp:73]: Done

File: silkworm/silkrpc/concurrency/context_pool.hpp (hunk 1 above)
Comment (Sixtysixter) [reply silkworm/silkrpc/concurrency/context_pool.hpp:133]: Done

**Review by canepat — APPROVED** [review]

**Pull request merged**